sencha-workspace/packages/slate-cbl/src/view/skill/OverviewBody.js

```diff
@@ -0,0 +1,92 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.skill.OverviewBody', {
+    extend: 'Ext.container.Container',
+    xtype: 'slate-cbl-skill-overviewbody',
+    requires: [
+        'Slate.cbl.model.Skill',
+
```

Comment:
when there are a lot of requirements we'll often use a blank line to separate them by top-level namespace. I don't think three justifies it in this case.

The blank line between requires and items though is standard practice as it separates the config options for Ext.define (documented here: http://docs.sencha.com/extjs/5.0/5.0.1-apidocs/#!/api/Ext.Class) and the rest which are new/overridden members for our new class.